### PR TITLE
Update spdx-license-list

### PR DESCRIPTION
**Pull request by frbuceta**
Fix test #7096

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by frbuceta** [comment]
@raymondfeng can you look at this?

File: packages/cli/generators/copyright/license.js [hunk 1]
```diff
@@ -40,8 +40,6 @@ This project is licensed under the ${license.name}, full text below.
 
 --------
 
-${license.name}
-
```
Comment (bajtos) [inline packages/cli/generators/copyright/license.js:44]: I find this slightly suspicious. Take a look at https://github.com/strongloop/loopback-next/blob/master/LICENSE, it starts with the following header:

```
Copyright (c) IBM Corp. 2017,2019.
Node module: loopback-next
This project is licensed under the MIT License, full text below.

--------

MIT license

(...)
```

I always thought the MIT license text is supposed to start with the license name?

File: packages/cli/test/integration/generators/copyright.integration.js [hunk 2]
```diff
@@ -271,7 +271,7 @@ notice and this permission notice appear in all copies.`,
     );
     assert.fileContent(
       path.join(sandbox.path, 'LICENSE'),
-      `MIT License Copyright (c) IBM Corp. ${year}
+      `Copyright (c) IBM Corp. ${year}
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
```
Comment (bajtos) [inline packages/cli/test/integration/generators/copyright.integration.js:276]:
```suggestion
      `Copyright (c) IBM Corp. ${year}

MIT license

Permission is hereby granted, free of charge, to any person obtaining a copy of
```

**Review by bajtos — COMMENTED** [review]
I think it would be better to replace `assert.fileContent` with snapshot-based checks, that way we can verify the full file content, not just small parts one by one.

**Comment by bajtos** [comment]
> I think it would be better to replace `assert.fileContent` with snapshot-based checks, that way we can verify the full file content, not just small parts one by one.

I started to work on this and realized that we cannot use snapshot-based tests here, because the license text is dynamic - it contains the current year.

**Comment by bajtos** [comment]
I find the current implementation and tests for `lb4 copyright` as difficult to reason about, let's improve them first so that we can get a better understand what is this pull request really changing. I opened a new PR for this improvement - see https://github.com/strongloop/loopback-next/pull/7117

File: packages/cli/generators/copyright/license.js (hunk 1 above)
Comment (frbuceta) [reply packages/cli/generators/copyright/license.js:44]: It is already integrated in the new version of this dependency

File: packages/cli/test/integration/generators/copyright.integration.js [hunk 3]
```diff
@@ -274,7 +273,7 @@ This project is licensed under the MIT License, full text below.
 
 MIT License
 
-MIT License Copyright (c) IBM Corp. 2021
+Copyright (c) IBM Corp. 2021
```
Comment (bajtos) [inline packages/cli/test/integration/generators/copyright.integration.js:276]: Awesome :+1:

**Review by bajtos — APPROVED** [review]
The new version looks great, thank you @frbuceta for taking care of handling the upgrade to a newer version of `spdx-license-list` ❤️

**Pull request merged**